sphinx_source/Ucuncu_taraf_moduller/django_modülü/django_2

```diff
@@ -0,0 +1,484 @@
+.. meta::
+   :description: Bu bölümde django APIsini öğreneceğiz.
```

Comment:
"API" kısaltmasını İngilizce'de okunduğu gibi okursak (ey-pi-ay) buradaki "APIsini", "API'ını" şekilde yazmamız gerekir.

Tek tek yazmadım ama belgenin devamında da yazım yanlışları var. Belgeye göz gezdirip onları düzeltebilir misiniz?

Reply:
Orada yanlış yazmışım. Ancak doğrusu şu olacak:"API'sını". Çünkü Türkçede İngilizce kelimeler yazılışına göre ek alıyorlar.

Reply:
Bu konuda pek bir bilgim yok. Haklı olabilirsiniz. Ancak okuyan kişiler bunu genelde İngilizce olarak okuyacaklardır ve bu durumda bu ek uygun olmuyor. Her ne kadar Türkçe kurallarına tam olarak uymak istesek de İngilizce'nin bu alandaki liderliğine bakarsak, bu pek mümkün değil. Ayrıca belgeleri okuyacak kişilerin akıcı okuyabilmelerini sağlamak, dört dörtlük Türkçe kullanmaktan bir tık daha önemli olmalı. Öyle olmasa, örneğin, "programlama" sözcüğü yerine çok bilinmeyen bir Türkçe karşılığını kullanıyor olurduk.